Sources/FoundationEssentials/FileManager/FileManager+Basics.swift

```diff
@@ -16,6 +16,28 @@ import Darwin
 import Glibc
 #elseif os(Windows)
 import CRT
+import WinSDK
+#endif
+
+#if os(Windows)
+extension _FILE_ID_128: Equatable {
+    public static func == (_ lhs: _FILE_ID_128, _ rhs: _FILE_ID_128) -> Bool {
+        return withUnsafePointer(to: lhs.Identifier.0) { pLHS in
+            let pLHSBuffer = UnsafeBufferPointer(start: pLHS, count: 16)
+            return withUnsafePointer(to: rhs.Identifier.0) { pRHS in
+                let pRHSBuffer = UnsafeBufferPointer(start: pRHS, count: 16)
+
+                return pLHSBuffer.elementsEqual(pRHSBuffer)
+            }
+        }
+    }
+}
+
+extension LARGE_INTEGER: Equatable {
```

Comment:
These will add public conformances to these types for all clients of `FoundationEssentials` - it seems like this might not be the right place for such an API. Should we downgrade these to `internal static` functions for now until we determine where the best place for such an API should be?

Reply:
Sure we can do that. I don't mind pushing this into the WinSDK overlay - the conformances are kinda nice. The problem is that we won't be able to use them until we get an updated snapshot.

Reply:
Sounds good, yeah if there's a better home for these we could add it there and add internal only functions for now, and then just switch over to the conformances once we get a newer snapshot